auto/cc/test

```diff
@@ -65,6 +65,8 @@ case $NXT_CC_NAME in
         # Do not export symbols except explicitly marked with NXT_EXPORT.
         NXT_CFLAGS="$NXT_CFLAGS -fvisibility=hidden"
 
+        NXT_CFLAGS="$NXT_CFLAGS -fno-strict-overflow"
```

Comment:
Repeated Cc field, and I'd also be happier if the commit message said that signed-integer overflow and pointer overflow are still not fair game.

Reply:
Anyway,

f3d7077b ("Compile with -fno-strict-overflow")

```
Reviewed-by: Alejandro Colomar <alx@kernel.org>
```

Reply:
Right, this is to cover existing cases, not that the we want to add more... I'll add words to that effect.